app/assets/javascripts/components/reward_page_components/StudentFinalRewardPage.jsx

```diff
@@ -3,17 +3,17 @@ class StudentFinalRewardPage extends React.Component {
     return (
       <div className='section'>
         <p className='flow-text'>
-          Heel erg bedankt dat je meedeed aan ons onderzoek! Door jouw deelname kunnen wij onze webapp zo verbeteren
-          dat deze veel beter zal aansluiten aan de wensen van toekomstige deelnemers. Zodra de gegevens bij ons
-          binnen zijn ontvangt jouw S-team begeleider jouw beloning en kan jij je beloning daar dus ophalen.
+          Heel erg bedankt voor je inzet voor dit onderzoek!
         </p>
         <p className='flow-text'>
-           In totaal heb je €{this.props.earnedEuros} verdiend.
+          In totaal heb je €{this.props.earnedEuros} verdiend.
```

Comment:
hadden we hier trouwens geen printasmoney functie ofzo voor tegenwoordig?

Reply:
heb je ook getest dat ze daadwerkelijk op deze pagina komen?

Reply:
Ja heb ik getest. Ze komen er alleen als ze tenminste 1 vragenlijst hebben ingevuld (maar goed, anders krijgen ze ook geen beloning).

Reply:
Fixed use of printAsMoney function.